### ISSUE TITLE
Validation loss always equals zero for Instance/Semantic Segmentation with Model Garden

### ISSUE THREAD

**Issue by luiz-felipe-moreira** [tensorflow/models#11117]
My experiments for instance segmentation output `'validation_loss': 0.0`, every time the metrics are summarized. It also happens in the official tutorials of instance/semantic segmentation.

I’m running the notebook [Instance Segmentation with Model Garden 1](https://www.tensorflow.org/tfmodels/vision/instance_segmentation) from Tensorflow Model Garden documentation. During the training,  it periodically displays the metrics, but the value of the validation loss is always zero. This can be seen in the output of the ‘Train and Evaluate’ section of the notebook.
For example:

```
...
eval | step:   1200 | steps/sec:    3.3 | eval time:   60.0 sec | output: 
{'AP': 0.0814979,
 'AP50': 0.16509584,
 'AP75': 0.07145937,
  ...
 'mask_ARs': 0.0015765766,
 'steps_per_second': 3.3340747415649394,
 'validation_loss': 0.0}
...
```

I noticed the same behavior in official Semantic Segmentation tutorial. In this tutorial, the outputs are shown on the tutorial page: [Semantic Segmentation with Model Garden](https://www.tensorflow.org/tfmodels/vision/semantic_segmentation#train_and_evaluate).

**Comment by laxmareddyp** [comment]
Hi @luiz-felipe-moreira,

Sorry for the delay in response, We are looking into this issue and will let you know once it is updated from our side.

Thanks.

**Comment by laxmareddyp** [comment]
Hi @luiz-felipe-moreira ,

Could you please check this [gist](https://colab.sandbox.google.com/gist/sineeli/a8a2c38546a7ad4402408826fffb6d2d/semantic_segmentation.ipynb) in which  `exp_config.task.validation_data.resize_eval_groundtruth = True` set this variable to true to enable validation on validation data for semantic segmentation and for instance segmentation the validation loss kept intentionally as zero.

Thanks.

**Comment by luiz-felipe-moreira** [comment]
Hi @laxmareddyp,

Thanks for you response.
I can't understand why the validation loss is kept intentionally as zero for instance segmentation. I think that tracking the value of the validation loss is necessary. 
For example, it's important to compare the graph of the validation loss with the graph of the training loss in order to recognize model overfitting. When you check the graphs in tensorboard, all graphs are shown perfectly, except for the validation loss graph.

**Comment by bharatjetti** [comment]
Hi @luiz-felipe-moreira 
In the tutorial for [Semantic segmentation](https://www.tensorflow.org/tfmodels/vision/semantic_segmentation#train_and_evaluate) I tried to run the notebook with latest TensorFlow version. I changed the code to get updated version of dataset and updated few api's to enable validation loss tracking. It is working. Please find the gist [here](https://colab.research.google.com/gist/bharatjetti/5d0bfc5902537d5ffe12ca86de151f74/semantic_segmentation.ipynb#scrollTo=WASJZ3gUH8ni).
